### PR TITLE
Document that Install Cloud SDK only works on Windows.

### PR DESCRIPTION
**Pull request by ILMTitan**
Issue brought to light by issue #56.

### CODE REVIEW

File: DETAILS.md [hunk 1]
```diff
@@ -43,7 +41,9 @@ The extension installs the following tasks:
   and user password for a [Google Compute Engine][ComputeEngine] virtual machine
   running windows.
   - **[Install Google Cloud SDK][install-cloud-sdk]**: Finds or installs the
-  [Google Cloud SDK][CloudSdk].
+  [Google Cloud SDK][CloudSdk] on windows.
```
Comment (iantalarico) [inline DETAILS.md]: Maybe "Windows only" at the start of the line?

File: DETAILS.md [hunk 2]
```diff
@@ -43,7 +41,9 @@ The extension installs the following tasks:
...
+  - **[Google Cloud SDK Tool Installer][cloud-sdk-tool]**: Installs the [Google Cloud SDK][CloudSdk] as a cached
```
Comment (iantalarico) [inline DETAILS.md]: Looks like the link ([Google Cloud SDK Tool Installer][cloud-sdk-tool]) doesn't work

File: DETAILS.md [hunk 3]
```diff
@@ -57,3 +57,4 @@ The extension installs the following tasks:
 [container-build]: https://github.com/GoogleCloudPlatform/google-cloud-tfs/blob/master/container-build-task/README.md
 [set-login-password]: https://github.com/GoogleCloudPlatform/google-cloud-tfs/blob/master/set-login-build-task/README.md
 [install-cloud-sdk]: https://github.com/GoogleCloudPlatform/google-cloud-tfs/blob/master/install-cloud-sdk-build-task/README.md
+[vsts-tool]: https://docs.microsoft.com/en-us/vsts/build-release/concepts/process/tasks#tool-installers
```
Comment (iantalarico) [inline DETAILS.md]: Same here

File: install-cloud-sdk-build-task/README.md [hunk 4]
```diff
@@ -10,6 +10,8 @@ and allows caching of Goolge Cloud SDK versions, but requires the  [VSTS Tool In
 
 ## Usage
 
+This task only works on build agents with PowerShell (windows build agents).
```
Comment (iantalarico) [inline install-cloud-sdk-build-task/README.md]: What do you think about "This task only works on Windows build agents"?  We can run Powershell on Unix now.

File: DETAILS.md (hunk 1 above)
Comment (ILMTitan) [reply DETAILS.md]: That is better. Thanks.

File: install-cloud-sdk-build-task/README.md (hunk 4 above)
Comment (ILMTitan) [reply install-cloud-sdk-build-task/README.md]: Done.

File: DETAILS.md (hunk 2 above)
Comment (ILMTitan) [reply DETAILS.md]: Fixed.

File: DETAILS.md (hunk 3 above)
Comment (ILMTitan) [reply DETAILS.md]: This link works fine for me.

**Comment by ILMTitan** [comment]
PTAL

File: DETAILS.md (hunk 3 above)
Comment (iantalarico) [reply DETAILS.md]: Opps I meant to delete this, sorry!

**Review by iantalarico — APPROVED** [review]
👍

**Pull request merged**